ocs_ci/ocs/ui/base_ui.py

```diff
@@ -941,6 +968,26 @@ def login_ui(console_url=None, username=None, password=None):
     driver.get(console_url)
     # Validate proceeding to the login console before taking any action:
     proceed_to_login_console()
+    if ibm_cloud_managed:
+        username_el = wait_for_element_to_be_clickable(login_loc["username"], 60)
+        username_el.send_keys(username)
+        continue_login_el = wait_for_element_to_be_clickable(
+            login_loc["continue_button"], 60
+        )
+        continue_login_el.click()
```

Comment:
just a recommendation: we use `do_click` and `get_elements`, they have imbedded logic on waits and screenshots